src/app/pages/storage/modules/pool-manager/store/pool-manager.store.ts

```diff
@@ -91,13 +104,20 @@ export const initialState: PoolManagerState = {
   },
 
   topology: initialTopology,
+
+  wizardRequiredStepsValidity: {
```

Comment:
I don't think we need to take data from components to be able to validate this. Name and settings used for data vdevs are already in state.